src/OrchardCore/OrchardCore.DisplayManagement/Views/ShapeResult.cs

```diff
@@ -288,6 +295,16 @@ public ShapeResult Cache(string cacheId, Action<CacheContext> cache = null)
             return this;
         }
 
+        /// <summary>
+        /// Sets a condition that must return true for the shape to render.
+        /// The condition is only evaluated if the shape has been placed.
+        /// </summary>
+        public ShapeResult IfAsync(Func<Task<bool>> ifAsync)
+        {
+            _ifAsync = ifAsync;
+            return this;
```

Comment:
Add new line before

Reply:
Don't really see why this is necesary @hishamco ? Have a look at the actual class.

It's not a hard and fast rule and it just wastes space in a builder like class like this.

Reply:
This is the format we use everywhere ;) no worry, the important thing is the name got pretty ;)